src/exportable/ExportableConfiguration.ts

```diff
@@ -26,6 +27,16 @@ export class ExportableConfiguration implements Exportable {
     if (this.config.version) {
       yaml.contents.add({ key: 'version', value: this.config.version });
     }
+    if (this.config.dependencies) {
+      const fshDependencies: any[] = [];
+      this.config.dependencies.forEach((dependency: ImplementationGuideDependsOn) => {
+        fshDependencies.push({
+          [dependency.packageId]: dependency.version
```

Comment:
The config file supports _two_ ways of indicating dependencies.  You implemented this approach:
```yaml
dependencies:
  hl7.fhir.us.core: 3.1.0
```

The other approach is:
```yaml
dependencies:
  hl7.fhir.us.core:
    id: uscore
    uri: http://hl7.org/fhir/us/core/ImplementationGuide/hl7.fhir.us.core
    version: 3.1.0
```

Usually we can derive the `uri` from the `packageId` (which is the `key` in the yaml), so the more advanced approach is usually only needed if the config had explicitly set an `id` for the dependency.

For GoFSH, I think that if `dependency.id` does not exist you can use `[dependency.packageId]: dependency.version` , but if `dependency.id` _does_ exist, you should probably use the more advanced format with `id` and `version` sub-keys (and maybe while you're at it go ahead and throw in the `uri`).

See: https://fshschool.org/docs/sushi/configuration/#recommended-configuration

Reply:
The other thing is that the value of `dependencies` should not be an array; it should be an object.  Currently you export this:
```yaml
dependencies:
  - hl7.fhir.us.core: 3.1.0
```
but it needs to be this:
```yaml
dependencies:
  hl7.fhir.us.core: 3.1.0
```

Reply:
Gotcha, I'll go ahead and implement these changes!